package.json

```diff
@@ -19,6 +19,7 @@
     "start-all:browser": "yarn start:server & yarn start:browser",
     "start-all:electron": "yarn start:server & yarn start:electron",
     "lint": "lerna run lint",
+    "lint:fix": "lerna run lint:fix",
```

Comment:
I was playing around with auto-fixing the linter because I normally have quite a few linting errors.  This script works, but I can remove it if there's a reason we don't do that.